src/pages/spawn/spawnHost/SpawnHostTableActions.test.tsx

```diff
@@ -0,0 +1,68 @@
+import { MockedProvider } from "@apollo/client/testing";
+import { GET_USER } from "gql/queries";
+import { act, fireEvent, render } from "test_utils/test-utils";
+import { CopySSHCommandButton } from "./SpawnHostTableActions";
+
+afterEach(() => {
+  jest.clearAllMocks();
+  jest.clearAllTimers();
+});
+
+const execCommand = jest.fn();
+const user = "bynn.lee";
+const hostUrl = "ec2-54-242-162-135.compute-1.amazonaws.com";
+
+test("CopySSHCommandButton", async () => {
+  const { queryByDataCy, queryByText } = render(
+    <MockedProvider mocks={[getUserMock]}>
+      <CopySSHCommandButton user={user} hostUrl={hostUrl} />
+    </MockedProvider>
+  );
+  await act(() => new Promise((resolve) => setTimeout(resolve, 0)));
+  jest.useFakeTimers();
+  document.execCommand = execCommand;
+  const copySshButton = queryByDataCy("copy-ssh-button");
+
+  // MouseEnter should trigger tooltip.
+  fireEvent.mouseEnter(copySshButton);
+  act(() => {
+    jest.runOnlyPendingTimers();
+  });
+  expect(queryByDataCy("copy-ssh-tooltip")).toBeInTheDocument();
+  expect(queryByDataCy("copy-ssh-tooltip")).toBeVisible();
+  expect(queryByText("Must be on VPN to connect to host")).toBeInTheDocument();
+
+  // Click should change tooltip message.
+  fireEvent.click(copySshButton);
+  expect(execCommand).toBeCalledWith("copy");
+  expect(queryByText("Copied!")).toBeInTheDocument();
+
+  // MouseLeave should cause tooltip to disappear.
+  fireEvent.mouseLeave(copySshButton);
+  act(() => {
```

Comment:
Can you add a comment explaining why this act is necessary.

Reply:
done